.gitignore

```diff
@@ -10,6 +10,7 @@ result
 .DS_Store
 .depend
 .cache
+proof-libs/fstar/rust_primitives/#*#
```

Comment:
Why do we ignore these?

Reply:
These are "autosave" files generated by Emacs